### ISSUE TITLE
docker build fails 

### ISSUE THREAD

**Issue by tasdikrahman** [gojekfarm/kingsly#24]
When running docker build, it fails

```
Step 2/10 : RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
 ---> Running in 24c4736f8eca
W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease  Unable to find expected entry 'main/binary-amd64/Packages' in Release file (Wrong sources.list entry or malformed file)
```

Reference build failure: https://travis-ci.org/gojekfarm/kingsly/builds/527203728

**Comment by tasdikrahman** [comment]
fixed in 839a12d05abaa7f0b01693ba2d1f322cfe6c18db

**Comment by tasdikrahman** [comment]
relevant thread https://superuser.com/q/1423486/366168
